proposals/TP1-xds-transport-next.md

```diff
@@ -398,7 +425,18 @@ the resource path. Continuing the previous example, this now looks like:
 
 Since each resource returned is subject to independent update via delta xDS and
 there is no explicit collection directory to update, glob collections are highly
-scalable.
+scalable. For example, consider a collection of 10k resources and the addition
+of a single resource. Glob collections will send the single additional resource. List
+collections requires that the collection directory (now containing 10001
+resource references) to be sent from server to client, as well as the additional
+new resource.
+
+As with list collections, context parameters in the request must be matched in
+responses. A request for
+`xdstp://some-authority/envoy.config.listener.v3.Listener/foo/*?some=thing` will
+match a resource
+`xdstp://some-authority/envoy.config.listener.v3.Listener/foo/bar?some=thing`
+but not `xdstp://some-authority/envoy.config.listener.v3.Listener/foo/bar`.
```

Comment:
So this makes sense to me once @markdroth explained that globs support delta (add/remove), but pedantically the text above and example do not make that clear. I understand you have it in examples below, but is there a way that you can make this section more clear that glob supports delta add/remove?